src/components/ViewWrapper.vue

```diff
@@ -212,7 +213,7 @@ export default class App extends Vue {
    * NOTE: This is only implemented for Alteration filings atm.
    * @param action the emitted action
    */
-  protected async handleFeeSummaryActions (action: FeeSummaryActions): Promise<void> {
```

Comment:
I removed a bunch of these because my IDE tools said the method (or property) wasn't visible in the template. Of course, the old code worked, but I didn't like seeing errors in my IDE.